docs/extensions/code-review-tool-attestations.md

```diff
@@ -0,0 +1,114 @@
+# Code Review Tool Attestations
+
+Last Modified: October 22, 2024
+
+gittuf's [reference authorization
+attestations](/docs/design-document.md#reference-authorization) can be used to
+meet a threshold of approvals for a change in a Git repository. Here, every
+approver signs the attestation themselves, minimizing the trusted entities
+during verification, but this has usability concerns. To balance usability and
+security, this extension proposes the use of special attestations created by
+popular code review tools.
+
+## Attestation Schema
+
+The attestation is similar to the reference authorization, but also includes
+information indicating the approvers.
+
+```
```

Comment:
Would the diff codeblock type be useful here to show what's new?

Reply:
This is a new predicate type altogether rather than an extension of the reference authorization type. I think that makes it easier to ensure we have no confusion in how these attestations are used.